index.html

```diff
@@ -1,13 +1,27 @@
 <!DOCTYPE html>
-<html>
-<head>
-	<title>JS Onboard Project</title>
-	<script type="text/javascript" charset="utf-8" src="third_party/jquery-2.0.3.min.js"></script>
-</head>
-
-<body>
-	<p>Hello</p>
-</body>
-
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Onboarding JavaScript Task</title>
+        <script type="text/javascript" charset="utf-8" src="third_party/jquery-2.0.3.min.js"></script>
+        <link rel="stylesheet" href="style.css" />
+        <script src="app.js" defer></script>
```

Comment:
You really should not be using `defer` in the project, but there are no rules against it, so I'm not going to ask you to remove it.